lib/unwrappr/git_command_runner.rb

```diff
@@ -50,10 +50,10 @@ def git_dir?
         git_wrap { !current_branch_name.empty? }
       end
 
-      def branch_created?
+      def branch_created?(base_branch:)
```

Comment:
An aside, I find it surprising that a `?` method causes a side-effect.

Reply:
Yeah, it's not a true predicate. I'll fix this separately.

Reply:
https://github.com/envato/unwrappr/pull/82